docs/envs/MaMuJoCo/ma_ant.md

```diff
@@ -40,7 +40,7 @@ If partitioning, is None then the environment contains a single agent with the s
 | 5   | Torque applied on the rotor between the front right two links     | -1          | 1           | angle_2 (front_right_leg)        | hinge | torque (N m) |
 | 6   | Torque applied on the rotor between the torso and back left hip   | -1          | 1           | hip_3 (back_leg)                 | hinge | torque (N m) |
 | 7   | Torque applied on the rotor between the back left two links       | -1          | 1           | angle_3 (back_leg)               | hinge | torque (N m) |
-### if partitioning == "2x4":  # neighbouring legs together
+### if partitioning == "2x4":  # neighboring legs together (front and back)
 ```{figure} Ant_2x4.png
```

Comment:
I currently can't visualize this image. Is it required?
https://robotics.farama.org/envs/MaMuJoCo/ma_ant/#if-partitioning-2x4-neighbouring-legs-together

Reply:
I added it, do no know why it was missing from repo.
I will be adding more images soonish ^ TM